bin/k8s_condor_cpu_efficiency.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Create a site for the default period for each of the cms types
+#
+# This script is intended to be used as cron job in kubernetes.
+
+# Kerberos
+keytab=/etc/condor-cpu-eff/keytab
+principal=`klist -k "$keytab" | tail -1 | awk '{print $2}'`
+echo "principal=$principal"
+kinit $principal -k -t "$keytab"
+if [ $? == 1 ]; then
+    echo "Unable to perform kinit"
+    exit 1
+fi
+
+currentDir=$(
+  cd "$(dirname "$0")" && pwd
+)
+if ! klist -s
+then
+    echo "There is no valid kerberos ticket yet"
+    kinit
```

Comment:
This will lead to interactive prompt for password which I think is wrong since it will break automation. Could you please adjust code to use only keytab.